### ISSUE TITLE
Add option to save traces during run_combos

### ISSUE THREAD

**Issue by elisabettai** [BlueBrain/BluePyMM#151]
Would it be possible to save the traces during run step? It can be useful to check the behaviour of different me-combos and to choose appropriate thresholds for model rejection. Maybe 'save_traces' can be specified as an option, with default as False (to avoid saving too many traces in case of large morphology releases).

**Comment by elisabettai** [comment]
Yes, I was using a temporary solution to save a plot of the responses.
Maybe saving the traces would be necessary, but I'm more thinking about a voltage collage (like in the issue https://github.com/BlueBrain/BluePyMM/issues/156)

**Comment by tapaswenipathak** [comment]
hello folks, is this ticket still open?
